chatcommands.py

```diff
@@ -1275,8 +1275,19 @@ def command_whois(message_parts, ev_user_id, wrap2, *args, **kwargs):
                                 wrap2.get_user(admin).last_message,
                                 wrap2.get_user(admin).last_seen)
                                for admin in admins_not_in_room]
+    singular_return_names = {"admin": "admin",
```

Comment:
This can be done more efficiently:

```python
return_names = { "admin": ["admin", "admins"], "code_admin": ["code admin", "code admins"] }
return_name = return_names[valid_roles[message_parts[1]][len(admin_ids) - 1]
```